cmd/vclusterctl/cmd/connect.go

```diff
@@ -76,8 +77,10 @@ func NewConnectCmd(globalFlags *flags.GlobalFlags) *cobra.Command {
 		Log:         log.GetInstance(),
 	}
 
+	_, validator := loftctlUtil.NamedPositionalArgsValidator(true, "VCLUSTER_NAME")
```

Comment:
since all of these use the same arg, we can use the same object initialized here: https://github.com/loft-sh/loft-enterprise/blob/f5dec01c83c2c5a2afd2f17392d2677edb2b5e59/staging/src/github.com/loft-sh/loftctl/v3/pkg/util/positional_args.go#L21C12-L21C12

Reply:
`loftctlUtil.VClusterNameOnlyValidator`

Reply:
Yup. Makes sense. Added.